platforms/index.rst

```diff
@@ -9,27 +9,27 @@ Supported Platforms
 Platform and board specific support is continually added to the SOF project as documented below.
 
```

Comment:
@mwasko what exactly is platform clock? Is the CPU freq?

Reply:
Its the timestamp we use for the trace. The userspace tools assume it's 19.2M (unless specified on the cmd line) since this was the time on the initial SOF platforms. It was causing a lot of confusion recently on some platforms that used a different clock...